### PR TITLE
Fixes related to sci backend

### LINKED ISSUE #354: Add real test for Payment use case using backend_sci
Add api test for payment backend: provider and requestor should have available deposit for making transactions.

### PR DESCRIPTION
**Pull request by kbeker**
Resolves #354

### CODE REVIEW

File: concent_api/api-integration-force-accept-or-reject-results-test.py [hunk 1]
```diff
@@ -271,6 +271,8 @@ def test_case_2b_not_enough_funds(cluster_consts, cluster_url, test_id):
                         task_id=task_id,
                         subtask_id=subtask_id,
                         deadline=current_time - (cluster_consts.subtask_verification_time),
+                        requestor_ethereum_public_key=b'0' * 64,
```
Comment (rwrzesien) [inline concent_api/api-integration-force-accept-or-reject-results-test.py]: Please extract `64` to constant.

File: concent_api/api-integration-force-payment.py [hunk 2]
```diff
@@ -49,25 +49,23 @@ def test_case_2d_send_correct_force_payment(cluster_consts, cluster_url, test_id
     correct_force_payment = force_payment(
         subtask_results_accepted_list=[
             subtask_results_accepted(
-                payment_ts=current_time - cluster_consts.payment_due_time - 10,
+                payment_ts=current_time - cluster_consts.payment_due_time - 30,
```
Comment (rwrzesien) [inline concent_api/api-integration-force-payment.py]: Please create utility function, as this calculation is being repeated several times.

File: concent_api/api-integration-force-payment.py [hunk 3]
```diff
@@ -221,26 +215,25 @@ def test_case_2_a_force_payment_with_subtask_result_accepted_where_ethereum_acco
...
+                    payment_ts=current_time - cluster_consts.payment_due_time - 30,
                     task_to_compute=create_signed_task_to_compute(
                         timestamp=parse_timestamp_to_utc_datetime(current_time),
                         task_id=task_id + 'b',
                         subtask_id=subtask_id + 'B',
                         deadline=current_time,
-                        requestor_ethereum_public_key="y" * 128,
-                        price=1000,
+                        price=15000,
+                        requestor_ethereum_public_key=b'0' * 64
```
Comment (rwrzesien) [inline concent_api/api-integration-force-payment.py]: Same here.

File: concent_api/api-integration-force-accept-or-reject-results-test.py (hunk 1 above)
Comment (kbeker) [reply concent_api/api-integration-force-accept-or-reject-results-test.py]: Done

File: concent_api/api-integration-force-payment.py (hunk 2 above)
Comment (kbeker) [reply concent_api/api-integration-force-payment.py]: Done

File: concent_api/api-integration-force-payment.py (hunk 3 above)
Comment (kbeker) [reply concent_api/api-integration-force-payment.py]: Done

**Pull request merged**